python/lsst/sims/maf/batches/skycoverage.py

```diff
@@ -0,0 +1,126 @@
+"""Evaluate some bulk properties of the sky coverage
+"""
+import numpy as np
+import lsst.sims.maf.metrics as metrics
+import lsst.sims.maf.slicers as slicers
+import lsst.sims.maf.metricBundles as mb
+from .colMapDict import ColMapDict
+
+__all__ = ['meanRADec', 'eastWestBias']
+
+def meanRADec(colmap=None, runName='opsim', extraSql=None, extraMetadata=None):
+    """Plot the range of RA/Dec as a function of night.
+
+    Parameters
+    ----------
+    colmap : dict, opt
+        A dictionary with a mapping of column names. Default will use OpsimV4 column names.
+    runName : str, opt
+        The name of the simulated survey. Default is "opsim".
+    extraSql : str, opt
+        Additional constraint to add to any sql constraints (e.g. 'night<365')
+        Default None, for no additional constraints.
+    extraMetadata : str, opt
+        Additional metadata to add before any below (i.e. "WFD").  Default is None.
+    """
+    if colmap is None:
+        colmap = ColMapDict('opsimV4')
+    bundleList = []
+    plotBundles = []
+
+    group = 'Open Shutter Fraction'
```

Comment:
Was the `group` here meant to be  `Open Shutter Fraction`? Maybe `Bulk Sky Coverage` or `East vs West`?